src/neptune/new/internal/utils/deprecation.py

```diff
@@ -22,16 +22,15 @@
 __all__ = ["deprecated", "deprecated_parameter"]
 
 
-def deprecated(*, alternative: Optional[str] = None, stack_level: int = 1):
+def deprecated(*, alternative: Optional[str] = None):
     def deco(func):
         @wraps(func)
         def inner(*args, **kwargs):
             additional_info = f", use `{alternative}` instead" if alternative else " and will be removed"
 
             warn_once(
                 message=f"`{func.__name__}` is deprecated{additional_info}."
-                f" We'll end support of it in `neptune-client==1.0.0`.",
-                stack_level=stack_level + 1,
+                f" We'll end support of it in `neptune-client==1.0.0`."
```

Comment:
Can we add to these a link to https://docs.neptune.ai/setup/neptune-client_1-0_release_changes ?